contrib/agent/src/main/java/io/opencensus/contrib/agent/AgentMain.java

```diff
@@ -51,8 +53,15 @@ private AgentMain() {
    * @see java.lang.instrument
    */
   public static void premain(String agentArgs, Instrumentation inst) throws Exception {
+    checkNotNull(inst);
```

Comment:
checkNotNull usually is better to use it with an error message because the log entry or exception may not be clear.

s/inst/instrumentation
s/checkNotNull(inst)/checkNotNull(instrumentation, "instrumentation")